notebooks/viz/metric-comparisons.ipynb

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
When bringing up the visualization, it looks like there are only variable names with values right under them. The rest is just empty space in the table. What is the intended visualization for this notebook?